igcollect/logfile_values.py

```diff
@@ -133,18 +133,17 @@ def get_distribution(self):
         return d
 
     def get_metric_value(self):
-
         if not self.values:
             return 0
 
         if not self.function:
             return float(self.get_last_value())
 
-        if 'distribution' in self.function:
+        if self.function == 'distribution':
             return getattr(self, 'get_' + self.function)()
 
-        if 'count_' in self.function:
-            if 'percentage' in self.function:
+        if self.function.startswith('count'):
+            if self.function.endswith('percentage'):
```

Comment:
Actually this also not completely right, as now it would also catch something like `count_something_percentage`, maybe rather compare it with equal direclty, so `self.function == 'count_percentage'.

Reply:
Btw, how like is it that more possible functions will be added later? If so, we should think of a better structure, like a `dict` with function name as key and function pointer as value.

Reply:
Actually it is count_200_percentage and so on and, actually, it could stay with "in" there is no sence in all those restrictions if we not properly raising errors. Nothing will happen if someone (only question why) will want to set ololocount instead of count.

Reply:
That's a good idea, let's do this. Although not in this PR as it is out of scope. But the arguments and function-checking really could need some redesign in general.